label_studio/core/settings/base.py

```diff
@@ -760,3 +760,5 @@ def collect_versions_dummy(**kwargs):
     'ssl_keyfile': get_env('REDIS_SSL_KEYFILE', None),
     'ssl_certfile': get_env('REDIS_SSL_CERTFILE', None),
 }
+
+OPENAI_API_VERSION = get_env('OPENAI_API_VERSION', '2024-06-01')
```

Comment:
this is specific to azure right? Can you rename to imply that?

Reply:
We are only using it for AzureOpenAI client at the moment but its not Azure specific. Could be used elsewhere as well